package.json

```diff
@@ -93,7 +95,11 @@
         "flow_deps"
     ],
     "jest": {
-        "resetMocks": false
+        "resetMocks": false,
+        "testTimeout": 10000,
+        "moduleNameMapper": {
+            "supabase-swr": "<rootDir>/node_modules/supabase-swr/dist/index.umd.js"
+        }
```

Comment:
We need to manually set this up otherwise jest cannot find this module... not totally sure why.